app/Filters/Tenant.php

```
@@ -0,0 +1,58 @@
+        'admin',
+        'api',
+        'docs',
+        'login',
+        'logout',
+        'register',
+        'auth',
+        'assets',
+        'static',
+    ];
+
+    public function before(RequestInterface $request, $arguments = null): null|ResponseInterface
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$arguments'. (undefined)

(UnusedFormalParameter)